lib/edgehog_device_forwarder/supervisors/caches.ex

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule EdgehogDeviceForwarder.Supervisors.Caches do
+  use Supervisor
+
+  @caches Application.compile_env!(:edgehog_device_forwarder, __MODULE__)
+
+  @spec all() :: keyword(atom())
+  def all, do: @caches
+
+  @spec all_ets_tables() :: [atom]
+  def all_ets_tables do
+    Keyword.values(@caches)
+  end
+
+  @spec for(module()) :: atom()
+  def for(module), do: Keyword.fetch!(@caches, module)
```

Comment:
`for` is slightly cursed due to it being a keyword, and it also doesn't express _what_ is returned. Change this to `cache_id_for`